value/value.go

```diff
@@ -187,7 +187,7 @@ func Equals(lhs, rhs Value) bool {
 	return true
 }
 
-// String returns a human-readable representation of the value.
+// ToString returns a human-readable representation of the value.
```

Comment:
That brings inconsistency. I suspect we could rename them `AsXXX`, like `AsString()`

Reply:
I just renamed the comment so golint is not complaining.
But I could open another PR renaming the functions.
Did you see my question regarding the naming in slack: https://kubernetes.slack.com/archives/C9VRU4PRB/p1576069598284000

Reply:
Yes, I agree that this was a poor choice for naming. I'd be happy to see a PR to rename all the `Type` methods into `AsType`.

Reply:
Can we keep this change for a different pull-request?

Reply:
It's still just the comment. But sure, I'll remove it.